freqtrade/freqai/spice_rack/lightgbm_config.json

```diff
@@ -0,0 +1,36 @@
+{
+
+    "freqai": {
+        "enabled": true,
+        "purge_old_models": true,
+        "train_period_days": 4,
+        "backtest_period_days": 2,
+        "identifier": "spicy-id",
+        "feature_parameters": {
+            "include_timeframes": [
+                "3m",
+                "15m",
+                "1h"
+            ],
+            "include_corr_pairlist": [
+                "BTC/USDT",
+                "ETH/USDT"
```

Comment:
i'm not sure this will work with a static config (at least not like this).

If i start "spice_rack" (without additional config, obviously) - i'd expect the corr_pairlist to actually work for my scenario.
If i'm now starting this on gateio futures (which doesn't have BTC/USDT - but BTC/USDT:USDT) - it'll fail as the informative pairs are not available.

Reply:
Fair point - do you see any way around this without asking the user to define exchange specific config params? Goal here is simplicity, so for me I would keep a set for all supported exchanges and call those accordingly.

Reply:
>  I would keep a set for all supported exchanges

for binance, that'd then be
BTC/USDT, BTC/BNB, BTC/ETH, BTC/BUSD, BTC/...  (obviously the same combinations with ETH or any other random pair).

You can't realistically maintain this as separate configs (that's 4 configs for binance spot alone - and we currently support 6 spot and 3 futures exchanges ... 

---

you need to do this dynamically i think (which shouldn't be too hard, either - as you should be able to filter`markets` - similar to the filters list-pairs / list-markets do (with both quote and base - obviously using stake-currency).

Reply:
I'm not sure I agree: why would I need to support *all* stake currencies for a corr_pairlist? All we need is the closest available two pairs to BTC/USDT and ETH/USDT. These are just corr_pairs - they don't need to follow any user stake or anything like that. 

That said, your solution is likely still the right one as we just need to filter the market and find closest string to those two. 

I'll have a look at implementing that.

Reply:
This should be handled with recent commit:

https://github.com/freqtrade/freqtrade/pull/7421/commits/ff300d5c85adcb59d1abfda99bf56e057a8a5950